modules/nodes-cluster-enabling-features-cli.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
+
+[id="nodes-cluster-enabling-features-cli_{context}"]
+= Enabling feature sets using the CLI
+
+You can use the OpenShift CLI (`oc`) to enable feature sets for all the nodes in a cluster by editing the `FeatureGate` custom resource (CR).
```

Comment:
s/enable feature sets for all the nodes/enable feature sets for all **of** the nodes/
or
/enable feature sets for all nodes/